pandas/core/reshape/merge.py

```diff
@@ -1323,25 +1321,27 @@ def _create_cross_configuration(
             cross_col,
         )
 
-    def _validate_specification(self) -> None:
+    def _validate_specification(self, left_on, right_on):
```

Comment:
Nit: While you're here, this function could use a better name like `_validate_left_right_on`

Reply:
good idea, updated